### DISCUSSION TITLE
What exactly do these options do?

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#5111] [Q&A] [answered]
Hi everyone! (for the one millionth time; It’s me again 😜)

today’s question: what exactly do these options do when you change them (RadioMaster MT12) I’ve played around with them before but I never saw any difference. (Picture credit to RadioMaster’s manual which gives credit to EdgeTX) 
![IMG_3171](https://github.com/EdgeTX/edgetx/assets/142270525/8fa3caef-5330-44d5-a39d-d251f06ff76c)

**Comment by pfeerick** [comment, marked as answer]
You usually only change them when you are changing the pot or switch hardware (hence why in RADIO->HARDWARE, duh!). Some settings will have effectively no effect, because the hardware doesn't let it do it. i.e. changing a 2POS to a momentary/toggle will probably do nothing (except prevent the switch from being shown in some places where a toggle switch is not suitable). For the MT12, you should only be interested in FL1 and FL2, in order to configure the addon module at the base of the grip. i.e. if you use the joystick module, you want S3/S4 configured as pots, and FL1/FL2 stay unassigned. If you are using the two switches, you want it as configured, i.e. S3/S4 as switches, and FL1/FL2 assigned to S3/S4, and the type of switches set. You should then have FL1 and FL2 as two additional switches you can use in the model settings.

**Reply by inventor7777** [reply]
Ooook. That makes sense! Thanks again!!
